public/app/core/components/NavBar/Next/NavBarNext.tsx

```diff
@@ -75,6 +82,8 @@ export const NavBarNext = React.memo(() => {
           }}
         >
           <FocusScope>
+            <div id={NAV_MENU_PORTAL_CONTAINER_ID} className={styles.menuPortalContainer} />
```

Comment:
Nit: I think would be nice if we are more explicit here, maybe we can create a new component called `<NavPortalContainer>` the same approach we have in `AppWrapper`?